src/python/library/build_wheel.py

```diff
@@ -103,7 +107,14 @@ def sed(pattern, replace, source, dest=None):
     print("Adding package files")
 
     mkdir(os.path.join(FLAGS.whl_dir, 'tritonclient'))
-    touch(os.path.join(FLAGS.whl_dir, 'tritonclient/__init__.py'))
+    cpfile('tritonclient/__init__.py',
```

Comment:
we use shutil.copyfile and shutil.copy elsewhere in the script - is it possible for us to bottom out on a single method?

Reply:
The shutil.copyfile requires the dest to be a file and it won't work with directories. That's why we are using two APIs.